SESSION_FACILITATOR_GUIDE.md

```diff
@@ -0,0 +1,31 @@
+# 💁🏽 Session Facilitator Guide
+
+This document is intended to provide a helpful starting point or reference for running sessions during summits.
+The new learnings and tools from a summit will ideally be PR'd in when each is concluded.
+
+🎉 Lists are included 🎉
+
+## 📅 in the days leading up to the session
+- [ ] If applicable, port the agenda into [tcq](https://tcq.app/)
+- [ ] Set a chair/facilitator for the session
+
+## 🛫 pre-flight checklist (when about to begin the session)
+- [ ]  If not pre-arranged, find consensus on a chair for the session. Scheduled sessions are likely to already have a facilitator/chair. Ad-hoc sessions will need someone to make sure everyone walks out of the room with actionable items.
+- [ ] If not pre-arranged, ask the table for volunteer note taker.
+- [ ] Have a person be take charge of each tool being used for the session. For a full list of tools see below.
+- [ ] Go to Zoom, start the call.
```

Comment:
What does one do if there are concurrent sessions and not enough zoom accounts?

Reply:
Thanks for raising this! 🙌🏽

Related comment: https://github.com/nodejs/summit/pull/113#discussion_r221643096

Who can help resolve a lack of zoom accounts?

Reply:
@nodejs/foundation anyone can direct Manil to the right person to talk to about this?

Reply:
@benjamingr There's a possible workaround [here](https://github.com/nodejs/summit/issues/111#issuecomment-426396414) for now